packages/graphql-server/src/errors/errors.ts

```diff
@@ -19,14 +16,42 @@ export const generateAuthenticationError = (traceId?: string) => {
 }
 
 export const generateUserInputError = (traceId?: string) => {
-  const error = new UserInputError(`${traceId || ''} - ${errorMessages.unprocessableEntity}`)
+  const error = new UserInputError(`${traceId || ''} - ${errorMessages.badRequest}`)
   logger.info('generateAuthenticationError', { traceId, error: JSON.stringify(error) })
   return error
 }
 
+export const generateValidationError = (traceId?: string) => {
+  const error = new ValidationError(`${traceId || ''} - ${errorMessages.badRequest}`)
+  logger.info('generateValidationError', { traceId, error: JSON.stringify(error) })
+  return error
+}
+
+export const generateForbiddenError = (traceId?: string) => {
+  const error = new ForbiddenError(`${traceId || ''} - ${errorMessages.forbidden}`)
+  logger.info('generateForbiddenError', { traceId, error: JSON.stringify(error) })
+  return error
+}
+
+export const generateInternalServerError = (traceId?: string) => {
+  const error = new ApolloError(`${traceId || ''} - ${errorMessages.internalServer}`, 'INTERNAL_SERVER_ERROR')
+  logger.info('generateInternalServerError', { traceId, error: JSON.stringify(error) })
+  return error
+}
+
+export const generateNotFoundError = (traceId?: string) => {
+  const error = new ApolloError(`${traceId || ''} - ${errorMessages.notFound}`, 'NOT_FOUND')
+  logger.info('generateNotFoundError', { traceId, error: JSON.stringify(error) })
+  return error
```

Comment:
I think we should `throw error` here instead of return, like this: 
<img width="763" alt="Screen Shot 2020-03-12 at 15 11 32" src="https://user-images.githubusercontent.com/26746668/76501065-30724300-6474-11ea-8a6a-9e24d8cd3a1a.png">

So that the `try/catch` will work correctly, and the `thrown errors` will be auto handled correctly by `apollo`

Reply:
Same for above errors

Reply:
@vuhuucuong actually I re-map errors to avoid show stack trace to user

Reply:
![image](https://user-images.githubusercontent.com/22660833/76502763-37e71b80-6477-11ea-8418-b5a77606b517.png)
this one is return error on the stack trace
![image](https://user-images.githubusercontent.com/22660833/76502802-4b928200-6477-11ea-809d-334e2231d68e.png)
because I use this function to remap and avoid show stack trace to the user